include/vcpkg/base/messages.h

```diff
@@ -767,8 +776,21 @@ namespace vcpkg
     DECLARE_MESSAGE(ExpectedFailOrSkip, (), "", "expected 'fail', 'skip', or 'pass' here");
     DECLARE_MESSAGE(ExpectedPortName, (), "", "expected a port name here");
     DECLARE_MESSAGE(ExpectedTripletName, (), "", "expected a triplet name here");
+    DECLARE_MESSAGE(ExportingPackage, (msg::package_name), "", "Exporting {package_name}...");
     DECLARE_MESSAGE(ExtendedDocumenationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
     DECLARE_MESSAGE(FailedToExtract, (msg::path), "", "Failed to extract \"{path}\":");
+    DECLARE_MESSAGE(FailedToFindPortFeature,
+                    (msg::value, msg::spec),
+                    "'{value}' is a port feature name.",
+                    "Could not find feature {value} in port {spec}.");
```

Comment:
```suggestion
                    "Could not find {value} in {spec}.");
```

I know the original code said feature and port here but I don't think it's necessary in this context.

Also consider reusing this message for https://github.com/microsoft/vcpkg-tool/blob/7d292feb55656816952b68efd4a1a738839d6c78/src/vcpkg/dependencies.cpp#L254-L259 (or using the language from there here too).

If there are more of these we could add `{feature}` or similar. (I wasn't opposed to adding `feature_spec` in our chat; I was only commenting that we didn't need it for that example)